src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1403,10 +1403,12 @@ public void LanguageVersion_CommandLineUsage()
             Assert.True(rangeEnd > rangeStart);
 
             string helpRange = help.Substring(rangeStart, rangeEnd - rangeStart).ToLowerInvariant();
-
+            var acceptableFollowingChar = new[] { '\r', '\n', ',' };
             foreach (var version in expected)
             {
-                Assert.True(helpRange.Contains(version), $"Missing version '{version}'");
+                var foundIndex = helpRange.IndexOf(version);
```

Comment:
It feels like we should make sure the previous character is a white-space, or a comma.